### PR TITLE
Mention security updates as driver

### PR DESCRIPTION
**Pull request by DavidSchinazi**

### CODE REVIEW

File: draft-iab-protocol-maintenance.md [hunk 1]
```diff
@@ -245,7 +245,10 @@ the negative effects on future implementations and the protocol as a whole.
 
 For a protocol to have sustained viability, it is necessary for both
 specifications and implementations to be responsive to changes, in addition to
-handling new and old problems that might arise over time.
+handling new and old problems that might arise over time. Additionally,
+since implementations now already benefit from periodic updates to mitigate
+security vulnerabilities, the cost of reacting to changes is much lower than it
+was in the past.
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:251]: I'm less sure about this.  It's a common argument I like to use, but I would want to inject a little more nuance into this.

When I've talked to people on the routing side of the IETF, they are adamant that the process for compatibility changes is very different to the one they use for security fixes.  In other words, it's not as simple as you make out (or at least people don't believe that it is that simple...).  Adding this text would add ammunition to their complaints about this draft.

Comment (DavidSchinazi) [reply draft-iab-protocol-maintenance.md:251]: That's totally fair. How about we don't mention security and just say that software update mechanisms have gotten better over the years making protocol reaction changes easier?

Comment (martinthomson) [reply draft-iab-protocol-maintenance.md:251]: That might be a better angle on this, yes.

Comment (DavidSchinazi) [reply draft-iab-protocol-maintenance.md]: Done

**Pull request merged**